src/Bicep.Core/Analyzers/Linter/Rules/UseLatestAzPowerShellVersionRule.cs

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Bicep.Core.CodeAction;
+using Bicep.Core.Diagnostics;
+using Bicep.Core.Semantics;
+using Bicep.Core.Syntax;
+using Bicep.Core.Text;
+using System.Globalization;
+
+namespace Bicep.Core.Analyzers.Linter.Rules
+{
+    public sealed class UseLatestAzPowerShellVersionRule : LinterRuleBase
+    {
+        public new const string Code = "use-latest-az-powershell-version";
```

Comment:
[nit]: Maybe this should be `recent` rather than `latest` to match the wording of the API version linter?
```suggestion
        public new const string Code = "use-recent-az-powershell-version";
```

Reply:
Agreed! Recent makes more sense. I have renamed in other places and file as well.